samples/drumthumper/src/main/cpp/DrumPlayerJNI.cpp

```diff
@@ -128,6 +128,27 @@ JNIEXPORT void JNICALL Java_com_google_oboe_sample_drumthumper_DrumPlayer_restar
     }
 }
 
+JNIEXPORT void JNICALL Java_com_google_oboe_sample_drumthumper_DrumPlayer_setPan(
+        JNIEnv *env, jobject thiz, jint index, jfloat pan) {
+    sDTPlayer.setPan(index, pan);
+}
+
+JNIEXPORT jfloat JNICALL Java_com_google_oboe_sample_drumthumper_DrumPlayer_getPan(
+        JNIEnv *env, jobject thiz, jint  index) {
+    // TODO: implement getPan()
```

Comment:
Can remove this TODO if you implemented `getPan`

Reply:
Done